ic-agent/src/agent/mod.rs

```diff
@@ -1328,6 +1345,17 @@ pub(crate) struct Subnet {
     canister_ranges: RangeInclusiveSet<Principal, PrincipalStep>,
 }
 
+/// API Boundary Node, which routes /api calls to IC replica nodes.
+#[derive(Debug)]
+pub struct ApiBoundaryNode {
+    /// Domain name
+    _domain: String,
```

Comment:
Presumably these are underscored to silence an 'unused' warning? They would also be silenced by marking the fields `pub`, which I think you have forgotten to do as users cannot do anything with `ApiBoundaryNode`.

Reply:
> Presumably these are underscored to silence an 'unused' warning?

Yes, exactly, changed to `pub`.